src/lib.rs

```diff
@@ -88,6 +88,37 @@
 //! At this time, we do not recommend use for security-critical applications.
 //! Use at your own risk.
 //!
+//! # 🔒 Networking
+//!
+//! The calling application is responsible for sending messages between
+//! participants in the protocol.
+//! All communication between [`Participant`]s must take place over a secure
+//! channel that satisfies mutual entity authentication and integrity.
+//!
+//! The protocol requires a UC-secure, synchronous, authenticated broadcast
+//! channel for use by the [`Participant`]s. Currently, the library handles this
+//! automatically by implementing the echo-broadcast protocol described by
+//! Goldwasser and Lindell[^echo]. This is the approach the paper recommends,
+//! but we note that this reduces the security of the protocol to selective
+//! abort[^abort] rather than the stronger notion of identifiable abort that a
+//! perfect authenticated broadcast protocol would allow.
```

Comment:
nits: 
- "note" ---> "emphasize" (or something stronger than "note")
-  delete "perfect", e.g., reword to "... that is achieved with an authenticated broadcast protocol"